src/coreclr/gc/gc.cpp

```diff
@@ -14208,12 +14209,15 @@ void allocator::thread_sip_fl (heap_segment* region)
             heap_segment_gen_num (region), heap_segment_mem (region), gen_number));
         // If we have a bucketed free list we'd need to go through the region's free list.
         uint8_t* region_fl_item = region_fl_head;
+        size_t total_free_size = 0;
         while (region_fl_item)
         {
             uint8_t* next_fl_item = free_list_slot (region_fl_item);
             thread_item (region_fl_item, size (region_fl_item));
+            total_free_size += size(region_fl_item);
```

Comment:
this `size` calculation is duplicated from the line above... should probably make a variable for it (`size_t s = size (region_fl_item)`)? it's most likely handled by CSE anyway but why not make it a bit more explicit :)

Reply:
Sure, I can do that. Probably won't make a difference in codegen though, because total_free_size is only used in the assert just after the loop, and so in release build, Ihope the compiler will be smart enough to eliminate the variable.